cfgmgr/vxlanmgr.cpp

```diff
@@ -282,6 +296,16 @@ bool VxlanMgr::doVxlanCreateTask(const KeyOpFieldsValuesTuple & t)
         // Suspend this message util the vrf is created
         return false;
     }
+    
+    // If the mac address has been set
+    auto macAddress = getVxlanRouterMacAddress();
+    if (macAddress.first)
```

Comment:
if (!macAddress.first) 

Typo? @Pterosaur

Reply:
Hi Wenda, Thanks for your comment. This is my fault.
I have solve this problem at : https://github.com/Azure/sonic-swss/pull/1142